### ISSUE TITLE
any alternatives or anything I can do to have you update this?

### ISSUE THREAD

**Issue by LucidEnemy** [hckr/fb-post-screenshot#37]
so I think with the way facebook has been going with comment and post removals this would be the best plug in ever. I admin a group and commonly get "called out" for removing things that very obviously would be either misinterpreted by facebook ML or actually against ToS. I really wanted this to work so I can automate it and remove as needed and have screenshot proof if or when someone messages about it. would really appreciate it if I can make this work some how. I know the layout hasnt changed too much after the large change. is there another project somewhere or anything I can do to have you work on this? I think it would really help the facebook community as a whole and really would help me :)

**Comment by hckr** [comment]
Hi, to be honest when I started developing this add-on, I had a lot more free time than now and that's why I didn't update it for a long time. Facebook was always changing some little details to prevent bots(?) from working correctly and than a major redesign happened. I was simply overwhelmed by the amount of changes between old and new layouts and gave up, especially given low interest in the add-on.

TL;DR I have much less time than I used to and Facebook frequently changed something in the layout which had caused the add-on to break in the past.

Since you're interested, I might look into the code on the weekend and then give feedback (here) whether I came up with any easy fixes.
